README.md

```diff
@@ -1 +1,118 @@
-# To-do-list
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./features/images/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Welcome to my Project</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 [To do list] <a name="about-project"></a>
+
+**[To do list]** is an application that allows users to add the list of activities they want to do. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+ 
+- **[HTML5]**
+- **[CSS3]**
+- **[Javascript]**
+
+### Key Features <a name="key-features"></a>
+
+- **[Adding and removing activities]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+-Clone the repository.
+-Create a directory in your device.
+
+### Prerequisites
+
+In order to run this project you need:
+-A code editor like VS code
+-Internet connection
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+  git clone https://github.com/Bupilipili/To-do-list.git
+ 
+  cd To-do-list
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [OPTIONAL] This project uses webpack to run, I would advise you to add commands needed to run the project such as:
```
npm install
npm start
```